test/e2e/rollout/rollout_test.go

```diff
@@ -0,0 +1,488 @@
+package rollout_test
+
+import (
+	"fmt"
+	"time"
+
+	api "github.com/flightctl/flightctl/api/v1alpha1"
+	"github.com/flightctl/flightctl/test/harness/e2e"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samber/lo"
+)
+
+var _ = Describe("Rollout Policies", func() {
+
+	const (
+		fleetName      = "rollout"
+		siteMadrid     = "madrid"
+		siteParis      = "paris"
+		siteRome       = "rome"
+		functionWeb    = "web"
+		functionDb     = "db"
+		labelSite      = "site"
+		labelFunction  = "function"
+		
+		defaultSleepTime = 30 * time.Second
+		setupSleepTime   = 2 * time.Minute
+		completionTimeout = 5 * time.Minute
```

Comment:
There is a default timeout in constants file https://github.com/flightctl/flightctl/blob/main/test/util/constants.go#L15